Describe Collector App global variables (#1858)
Kong/docs.konghq.com: app/enterprise/1.3-x/brain-immunity/install-configure.md

3. **with-redis.yml** - creates redis instance that collector app will use if redis instance not already provided.

##### Collector App Environment Variables
The Collector App relies on several environment variables that need to be configured well for proper function.  These are the most impactful environment variables that should be specified to match the overall Kong and deployment environment, and what impact the variable has on Collector App.

> These variables have defaults in the provided docker-compose.yml that assume deployment with docker-compose on the same network that Kong is deployed on.  However, if this is not the case for your planned deployment of Collector App, please adjust the variables in the docker-compose.yml or specify the value of the varaibles docker-compose up like ```ENVVAR=somevalue ENVVAR=anothervalue docker-compose up```

Reviewer: ```suggestion
The provided `docker-compose.yml` file has these variables set to defaults that assume deployment with `docker-compose` on the same network that Kong is deployed. However, if this is not the case for your planned deployment of Collector App, please adjust the variables in the `docker-compose.yml` file, or specify the values of the variables with `docker-compose up`:
```ENVVAR=<somevalue> ENVVAR=<anothervalue> docker-compose up```
```

> 1. **KONG_PROTOCOL**: The protocol that Kong Admin API can be reached at.  It's going to be either http or https.

Reviewer: ```suggestion
1. **KONG_PROTOCOL**: The protocol that the Kong Admin API can be reached at. The possible values are http or https.
```

> 2. **KONG_HOST**: The host name that Kong Admin API can be reached at.  If deploying with Docker Compose, this is going to be the name of the Kong container specified in the compose file.  If Kong Admin has been exposed behind a web address, KONG_HOST will be that web address.

Reviewer: ```suggestion
2. **KONG_HOST**: The hostname that the Kong Admin API can be reached at. If deploying with Docker Compose, this is the name of the Kong container specified in the compose file. If Kong Admin has been exposed behind a web address, KONG_HOST must be that web address.
```

> 3. **KONG_PORT**: The port where Kong Admin can be found.  Together with KONG_PROTOCOL and KONG_HOST, Collector App is able to communicate with Kong Admin

Reviewer: This sentence sounds like it cuts off short - I'm not sure if there's supposed to be something else here? If not, then maybe rephrase to:  

Collector App requires this setting, along with KONG_PROTOCOL and KONG_HOST, to communicate with Kong Admin.

4. **KONG_ADMIN_TOKEN**: The authentication token used to validate requests for Kong Admin API, if configured.

> 5. **SQLALCHEMY_DATABASE_URI**: The sql alchemy formated URI that points to the Postgres Database that collector App is using as a backend.  The format is: postgresql://<USER>:<PASSWORD>@<POSTGRES-HOST>:<POSTGRES-PORT>/collector

Reviewer: ```suggestion
5. **SQLALCHEMY_DATABASE_URI**: The SQLAchemy formatted URI that points to the Postgres Database that Collector App is using as a backend. The format is: `postgresql://<USER>:<PASSWORD>@<POSTGRES-HOST>:<POSTGRES-PORT>/collector`.
```

> 6. **SLACK_WEBHOOK_URL**: The url of the slack channel that Immunity alert notifications should be sent to.  This url will be the default channel alerts are sent to, however more channels and rules configuring which alerts to send to which channel can be made.  See the [Adding a Slack Configuration](#adding-a-slack-configuration)

Reviewer: ```suggestion
6. **SLACK_WEBHOOK_URL**: The url of the Slack channel that Immunity alert notifications should be sent to. This url will be the default channel for alerts, but you can also add more channels and rules configuring which alerts to send to which channel. See [Adding a Slack Configuration](#adding-a-slack-configuration).
```
